mopidy/compat.py

```diff
@@ -62,6 +63,13 @@ def itervalues(dct, **kwargs):
 
     from itertools import zip_longest  # noqa
 
+    import inspect  # noqa
+
+    def getargspec(func):
```

Comment:
Please make use of this in `mopidy/mpd/protocol/__init__.py` too, where I've already did some jumping through hoops to make the code work with both `getargspec()` and `getfullargspec()`.